### ISSUE TITLE
no way to install the package properly

### ISSUE THREAD

**Issue by mikkokotila** [Esukhia/bophono#7]
- pypi does not work at all
- If install from git, then the following error will come: 

`FileNotFoundError: [Errno 2] No such file or directory: '/Users/upstairs/opt/miniconda3/envs/wip/lib/python3.8/site-packages/bophono/data/chinese/zhuyin.csv'`

There should not be static files as dependencies for packages unless there is an exceptional reason for it. When that is the case, the data should not be inside the package,  but pulled from github or other repository.

**Comment by eroux** [comment]
Good idea yes, the csv should be transformed into a Python file, PRs welcome!

**Comment by eroux** [comment]
can you try the 0.1.1 release on https://pypi.org/project/bophono/0.1.1/ ?

**Comment by eroux** [comment]
(sorry, 0.1.2)
